app/service/parsing_svc.py

```diff
@@ -1,14 +1,48 @@
-import plugins.stockpile.parsers.standard as parsers
-import plugins.stockpile.parsers.mimikatz as mimikatz_parser
+import glob
 from base64 import b64decode
+from pydoc import locate
 
 from app.service.base_service import BaseService
 
 
 class ParsingService(BaseService):
 
     def __init__(self):
+        self.parsers = dict()
         self.log = self.add_service('parsing_svc', self)
+        
+    async def load_parser(self, p_id, name, parser):
+        """
+        Load the specified parser plugin
+        :param p_id:  the id of the yaml file that defines the parser
+        :param name:  the unique name of the parser
+        :param parser:  an instance of the parser class 
+        :return: None
+        """        
+        if name in self.parsers:
+            self.log.warning('Duplicate parser name detected:  %s:%s' % (p_id, name))
+        else:
```

Comment:
function name is a bit misleading as load_parser and load_parsers do different things
this function looks to be adding the parser but not really loading anything